src/media/entities/media.repository.ts

```diff
@@ -10,4 +12,19 @@ export class MediaRepository extends Repository<Media> {
     newFile.name = file.originalname;
     return await this.save(newFile);
   }
+
+  getFullFileName(id: string): string {
```

Comment:
This method's logic have nothing to do with database. Accordingly, it should be added in the service instead

Reply:
My point of view, since repository is meant to deal with our storage media, so whether its database or file system as the case here, this should be contacted through the repository, since I do need to access my storage media which is local system here, to search for the presence of a file, exactly as we do if we had a database instead of storing to local system (this will be easier to solve in milestone 2)